english/play-some-cricket/playsomecricket.md

```diff
@@ -0,0 +1,6 @@
+[Checkout most rare and funny moments incricket history.](https://youtu.be/fzqh6_90akI)
```

Comment:
I just added the link to the new file and commit-ed it.

Please review this, @casey-collab